packages/zowe-explorer/CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ All notable changes to the "vscode-extension-for-zowe" extension will be documen
 
 - Update Zowe SDKs to `8.2.0` to get the latest enhancements from Imperative.
 - Added expired JSON web token detection for profiles in each tree view (Data Sets, USS, Jobs). When a user performs a search on a profile, they are prompted to log in if their token expired. [#3175](https://github.com/zowe/zowe-explorer-vscode/issues/3175)
+- Add Data Set or USS resources to a virtual workspace with the new "Add to Workspace" context menu option. [#3265](https://github.com/zowe/zowe-explorer-vscode/issues/3265)
```

Comment:
- "Data Set" (or "Data Sets") should be lowercase
- We should say either "Add a data set ... " or "Add data sets ... "

Reply:
Resolved in 7f0dfc9